CactbotOverlay/FFXIVProcess.cs

```diff
@@ -570,10 +485,19 @@ public class EntityData {
         // The pointer can be null when not logged in.
         return null;
       }
-
-      return Read8(job_inner_ptr, kJobDataInnerStructSize);
+      job_inner_ptr = IntPtr.Add(job_inner_ptr, kJobDataInnerStructOffset);
+      return Read8(job_inner_ptr, kJobDataInnerStructSize - kJobDataInnerStructOffset);
```

Comment:
Can you subtract this from the size in the declaration rather than here at the point of use?

Reply:
Done.